lib/common/util.js

```diff
@@ -138,13 +138,25 @@ exports.publishChange = publishChange;
 
 
 /*
- * Poll a job until it reaches either the succeeded or failed state.
+ * Poll a job until it:
+ * - reaches either the succeeded or failed state
+ * - completes a given task
+ *
+ * 'opts' is an options object whose properties can be:
+ * - 'taskName': a string that identifies the nane of the task to wait for
+ * before calling the callback 'cb'. If 'opts.taskName' is set, then pollJob
+ * will not wait for the whole job to succeed or fail, but instead will just
+ * wait for that specific task to complete.
  *
  * Note: if a job fails, it's the caller's responsibility to check for a failed
  * job.  The error object will be null even if the job fails.
  */
 
 function pollJob(opts, cb) {
+    assert.object(opts, 'opts must be an object');
```

Comment:
Right, absolutely, I found that out recently but I hadn't updated this code, thanks!